pkg/metamorph/client_test.go

```diff
@@ -58,6 +58,33 @@ func TestClient_SetUnlockedByName(t *testing.T) {
 	}
 }
 
+func TestStatusWaiting(t *testing.T) {
```

Comment:
Would be nice if this test could be just another test case in `TestClient_SubmitTransaction`